f5/bigip/test/test___init__.py

```diff
@@ -42,3 +50,8 @@ def test___get__attr(FakeBigIP):
     with pytest.raises(AttributeError):
         FakeBigIP.this_is_not_a_real_attribute
     assert FakeBigIP.hostname == 'FakeHostName'
+
+
+def test_meta_data_uri(FakeBigIPWithPort):
```

Comment:
This is slightly more specific than a just a test of "meta_data_uri" right, it's a test of the non-default port number, so let's rename it to something that includes the words "port", "number", and "default".